### DISCUSSION TITLE
Who are the dev team?

### DISCUSSION THREAD

**Discussion by marcosscriven** [Blackymas/NSPanel_HA_Blueprint#247] [General]
I see both in the issues and discussion areas talk of “we”. Interested who are the team here?

**Comment by Blackymas** [comment]
hi,

I started alone. Then @deejaybeam joined the team. 

But in the meantime we have more members in the team

- Code, issue, Wiki -> @danwright87 

- issue, code -> @hollabee 


There was also already some help or improvements directly from the community

You can see them all on the Github start page on the right side under Contributors ;)

**Reply by marcosscriven** [reply]
Interesting - I guess what I really meant is this a hobby/project by individuals, or is it sponsored by a company?

Awesome work, by the way!

**Reply by Blackymas** [reply]
thanks - it is a hobby ;)
